fda-model/structures/DeterministicStructure.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace structures
+{
+    public class DeterministicStructure
+    {
+        private int _fdid;
+        private double _foundationHeight;
+        private double _StructureValue;
+        private double _ContentValue;
```

Comment:
will content value be computed using a sampled CSVR as part of sampling the structure inventory?

Reply:
Right now it's assuming that's all available in the structure inventory. I'd be tempted to say that our economists should apply those kinds of factors when they create the structure inventory, rather than us try to handle it behind the scenes, but I wont die on that hill. I'm just lazy

Reply:
FINE. ADDED.